app/controllers/api/v1/users/wines_controller.rb

```diff
@@ -1,6 +1,10 @@
 class Api::V1::Users::WinesController < ApplicationController
   def create
+    # wine = Wine.find_or_create_by(api_id: params[:wine_id]) do |wine|
+    #   wine.name = params[:name]
+    # end
```

Comment:
Do we want to leave this commented code just in case something breaks for now? Or should we remove it?